src/main/java/org/prebid/server/auction/ExchangeService.java

```diff
@@ -608,7 +608,8 @@ private static BidRequestCacheInfo bidRequestCacheInfo(ExtRequestTargeting targe
     private static TargetingKeywordsCreator buildKeywordsCreator(ExtRequestTargeting targeting, boolean isApp) {
         return targeting != null
                 ? TargetingKeywordsCreator.create(parsePriceGranularity(targeting.getPricegranularity()),
-                targeting.getIncludewinners() != null ? targeting.getIncludewinners() : true, isApp)
+                targeting.getIncludewinners() != null ? targeting.getIncludewinners() : true,
+                targeting.getIncludebidderkeys() != null ? targeting.getIncludebidderkeys() : true, isApp)
```

Comment:
Maybe we do not need to check includeWinners and IncludeBidderKeys for null, because for now, if targeting is not null, these fields will be populated with default values in AmpRequestFactory and AuctionRequestFactory.